uniswap-v2/Makefile

```diff
@@ -1,7 +1,16 @@
+.PHONY: build_all
+build_all:
```

Comment:
erc20-holdings and maybe some other substreams in our repo have the same erc20-price dependency. Would you mind modifying their Makefiles to have the same cmds like this one also please?